pages/establishment/pdf/views/establishment.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import map from 'lodash/map';
+import StandardConditions from '../../../common/views/pdf/standard-conditions';
+
+const getName = person => `${person.firstName} ${person.lastName}`;
+
+const triggerContinued = () => {
+  let first = true;
+
+  return () => {
+    if (first) {
+      first = false;
+      return '';
+    }
+    return ' (continued)';
+  };
+};
+
+const Establishment = ({ establishment, content }) => {
+  const licenceHolder = establishment.pelh;
+
+  const hasAdditionalConditions = !!establishment.conditions;
+  const hasApprovedAreas = !!establishment.places;
+  const hasAuthorisations = establishment.authorisations && establishment.authorisations.length > 0;
+  const killing = establishment.authorisations.filter(({ type }) => type === 'killing');
+  const rehomes = establishment.authorisations.filter(({ type }) => type === 'rehomes');
+
+  const continued = triggerContinued();
+
+  return (
+    <Fragment>
+      <div className="logo"></div>
+
+      <h3 className="licence">ESTABLISHMENT LICENCE</h3>
+
+      <h1 className="establishment-name">{establishment.name}</h1>
+      <ReactMarkdown>{establishment.address}</ReactMarkdown>
+
+      <h4>Licenced to carry out:</h4>
+      <ul>
+        {
+          ['procedure', 'breeding', 'supplying'].filter(auth => establishment[auth]).map(auth =>
+            <li key={auth}>{content.licenced[auth]}</li>
+          )
+        }
+      </ul>
+
+      <section className="licence-holder">
+        <h2>Establishment licence holder</h2>
+        <p><strong>{`${licenceHolder.firstName} ${licenceHolder.lastName}`}</strong></p>
```

Comment:
The PEL holder only exists for some establishments. Others have no PEL holder and an NPRC instead.

Need to ensure this works ok for those establishments as well.